Source/Core/DolphinQt2/Translation.h

```diff
@@ -0,0 +1,7 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+void InitializeTranslation();
```

Comment:
You could provide a namespace around this so it be something like

```cpp
Translations::Initialize();
```

at the call site, but feel free to ignore this.

Reply:
Oh, so the namespace matches the filename? That's a good idea. Done.